lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu 
+  
+  attr_reader :menu
+  
+  def initialize
+    @menu = [
```

Comment:
Potential to use the 'Open Closed Principle' instead of hardcoding the items in the menu, as suggested here: https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md